moldock/preparation_for_docking.py

```diff
@@ -121,26 +119,9 @@ def add_protonation(db_fname):
         conn.close()
 
 
-def mk_prepare_ligand_string(molecule_string, build_macrocycle=True, add_water=False, merge_hydrogen=True,
-                             add_hydrogen=False, pH_value=None, verbose=False, mol_format='SDF'):
-
-    mol = obutils.load_molecule_from_string(molecule_string, molecule_format=mol_format)
-
-    if pH_value is not None:
-        mol.CorrectForPH(float(pH_value))
-
-    if add_hydrogen:
-        mol.AddHydrogens()
-        charge_model = ob.OBChargeModel.FindType("Gasteiger")
-        charge_model.ComputeCharges(mol)
-
-    m = Chem.MolFromMolBlock(molecule_string)
-    amide_rigid = len(m.GetSubstructMatch(Chem.MolFromSmarts('[C;!R](=O)[#7]([!#1])[!#1]'))) == 0
-
-    preparator = MoleculePreparation(merge_hydrogens=merge_hydrogen, macrocycle=build_macrocycle,
-                                     hydrate=add_water, amide_rigid=amide_rigid)
-                                     #additional parametrs
-                                     #rigidify_bonds_smarts=[], rigidify_bonds_indices=[])
+def mk_prepare_ligand_string(mol, verbose=False):
```

Comment:
rename this function according to the changes - `mk_prepare_ligand`
please do not forget to apply the same changes in other modules, if necessary